cloudpathlib/gs/gsclient.py

```diff
@@ -169,15 +169,7 @@ def _is_file_or_dir(self, cloud_path: GSPath) -> Optional[str]:
                 return None
 
     def _exists(self, cloud_path: GSPath) -> bool:
-        # short-circuit the root-level bucket
-        if not cloud_path.blob:
-            try:
-                next(self.client.bucket(cloud_path.bucket).list_blobs())
-                return True
-            except NotFound:
-                return False
-
-        return self._is_file_or_dir(cloud_path) in ["file", "dir"]
+        return self.client.bucket(cloud_path.bucket).exists()
```

Comment:
The `_exists()` is for all potential paths, not just buckets so I think the update is just for the short-circuit path of checking if the bucket exists.

Should be something like:

```python
if not cloud_path.blob:
    return self.client.bucket(cloud_path.bucket).exists()
    
return self._is_file_or_dir(cloud_path) in ["file", "dir"]
```